styles/home.css

```diff
@@ -186,9 +186,7 @@
   .logotext {
     display: none;
```

Comment:
Just a suggestion, for consistent naming of classes, you could separate words using - or _ would look more consistent and easy to distinguish like its an image or text element.

Reply:
I'll look upon your suggestions Thanks.

Reason behind changing the header on authentication page.

1. User is already on the authentication page so there is no need to put the login and signup button over there.
2. I am doing flex-center so there will be no more white spaces in header elements.

Reply:
So you have done it intentionally, that's ok. Btw beautiful landing page design.